rust/signed_doc/tests/common/mod.rs

```diff
@@ -28,15 +29,18 @@ pub fn test_metadata() -> (UuidV7, UuidV4, serde_json::Value) {
     (uuid_v7, uuid_v4, metadata_fields)
 }
 
-pub fn create_dummy_key_pair() -> anyhow::Result<(
+pub fn create_dummy_key_pair(
+    role_index: Option<RoleIndex>,
```

Comment:
I think it would be better to always provide a `RoleIndex` value, so it would be more explicit on the test case implementation how our document is generated.
```suggestion
    role_index: RoleIndex,
```